nightwatch.config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  specsDir: './test/e2e/nightwatch',
+  commandsDir: './test/e2e/nightwatch/commands',
+  runner: 'nightwatch',
+  testSettings: {}
```

Comment:
This is really just here as a placeholder in case we want to move some test settings or CLI arguments to the configuration file.  Most of the options are either made redundant or obviated by `get-test-files`, but we can add them in later if we'd like.